### DISCUSSION TITLE
What is your favourite movie ?

### DISCUSSION THREAD

**Discussion by 3207-Rhims** [NAIST-SE/AttractingContributors#1161] [2024 Attraction Contributions]
My Repo - https://github.com/3207-Rhims/AttractingContributors-Movie/

**Comment by ryosuke-sota** [comment]
I just made a PR on your repo. </br>Could you contribute to mine? </br>Here : [my project link (fav drinks)](https://github.com/ryosuke-sota/AttractingContributors.git)  </br>Thank you.

**Comment by kohanaist** [comment]
I just made a PR on your repo. 
Could you contribute to mine? 
https://github.com/kohanaist/AttractingContributors-technical-or-specialized-book

**Comment by r-nishioka** [comment]
I just made a PR on your repo. 
Could you contribute to mine? 
[Where would you like to trip in Japan?](https://github.com/r-nishioka/AttractingContributors-Trip.git)

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1148

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git

**Comment by kally1218** [comment]
I updated my pull request.
Could you kindly help to contribute my 🔗 https://github.com/kally1218/AttractingContributors
Thanks so much! 😭
